### PR TITLE
Update right nav bar and log position

### PR DESCRIPTION
**Pull request by Lijiaoa**
image
image
image
image

image
image
image
image
image

### CODE REVIEW

File: ts/webui/src/static/function.ts [hunk 1]
```diff
@@ -262,6 +262,13 @@ function formatComplexTypeValue(value: any): string | number {
     }
 }
 
+function caclMonacoEditorHeight(height): number {
+    // [Search space 56px] + [marginBottom 18px] +
+    // button[height: 32px, marginTop: 45px, marginBottom: 7px]
+    // panel own padding-bottom: 20px;
+    return height - 178;
```
Comment (colorjam) [inline ts/webui/src/static/function.ts:269]: why absolutely subtract 178 ?

Comment (Lijiaoa) [reply ts/webui/src/static/function.ts:269]: line266-268 notes that.

**Comment by colorjam** [comment]
Can we use more impressive icons for `Search Space` and `Config` ?

File: ts/webui/src/static/function.ts (hunk 1 above)
Comment (colorjam) [reply ts/webui/src/static/function.ts:269]: got it

**Comment by Lijiaoa** [comment]
> Can we use more impressive icons for `Search Space` and `Config` ?

had updated new icon for `Search space` and `Config`
image
image
image
image

**Review by kvartet — APPROVED** [review]

**Review by colorjam — APPROVED** [review]

**Review by liuzhe-lz — APPROVED** [review]

**Pull request merged**